op-batcher/batcher/channel_builder_test.go

```diff
@@ -490,7 +504,7 @@ func TestBuilderAddBlock(t *testing.T) {
 	require.NoError(t, err)
 
 	// Add a nonsense block to the channel builder
-	err = addNonsenseBlock(cb)
+	err = addMiniBlock(cb)
 	require.NoError(t, err)
```

Comment:
Can inline all of these to be more idiomatic

```suggestion
	require.NoError(t, addMiniBlock(cb))
```

Reply:
I agree and also do it this way. This particular diff here is not in a test I wrote but someone else. I just renamed the function `addNonsenseBlock` to `addMiniBlock` in this commit and that's where this diff comes from. I didn't want to further blow up this PR by refactoring other tests. So I'll leave it at this. Next time this test's content is touched, the author should fix it.